### PR TITLE
[unit-tests] added unit tests for OcNotificationMessage component

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Added unit tests for `OcNotificationMessage` component

## Related Issue

- Part of #1366 

## How Has This Been Tested?

- 👨🏼‍🎨 
- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation added/updated

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by individual-it — COMMENTED** [review]
please provide some more details about why the changes in the code are needed

File: src/components/OcNotificationMessage.vue [hunk 1]
```diff
@@ -3,26 +3,31 @@
     <oc-icon :variation="iconVariation" size="large" name="info" class="oc-mr-s" />
     <div
       class="uk-flex uk-flex-wrap uk-flex-middle uk-flex-1 oc-mr"
-      :role="status === 'danger' ? 'alert' : 'status'"
-      :aria-live="status === 'danger' ? 'assertive' : 'polite'"
+      :role="role"
```
Comment (kiranparajuli589) [inline src/components/OcNotificationMessage.vue:6]: avoid using logic in the template, rather use computed section

**Review by saw-jan — COMMENTED** [review]
LGTM

File: src/components/OcNotificationMessage.spec.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      }
+    )
+    it("should set status as icon variation", () => {
+      const wrapper = getWrapper({ status: "primary" })
+      const iconElement = wrapper.find("oc-icon-stub")
+      expect(iconElement.exists()).toBeTruthy()
+      expect(iconElement.attributes("variation")).toBe("primary")
+    })
+    describe("role and aria live of message content wrapper", () => {
+      it("if status is not danger", () => {
```
Comment (LukasHirt) [inline src/components/OcNotificationMessage.spec.js:66]: I am kinda missing here what the step should actually do. If true then what?

File: src/components/OcNotificationMessage.spec.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        const wrapper = getWrapper({})
+        const messageContentEl = wrapper.find(selectors.messageWrapper)
+        expect(messageContentEl.attributes("role")).toBe("status")
+        expect(messageContentEl.attributes("aria-live")).toBe("polite")
+      })
+      it("if status is danger", () => {
```
Comment (LukasHirt) [inline src/components/OcNotificationMessage.spec.js:72]: I am kinda missing here what the step should actually do. If true then what?

File: src/components/OcNotificationMessage.spec.js (hunk 2 above)
Comment (kiranparajuli589) [reply src/components/OcNotificationMessage.spec.js]: fixed

File: src/components/OcNotificationMessage.spec.js (hunk 3 above)
Comment (kiranparajuli589) [reply src/components/OcNotificationMessage.spec.js]: fixed

**Review by LukasHirt — APPROVED** [review]
LGTM

**Pull request merged**